Goobi/src/de/sub/goobi/beans/Prozess.java

```diff
@@ -466,7 +466,7 @@ public Boolean getTifDirectoryExists() {
 		if (testMe.list() == null) {
 			return false;
 		}
-		if (testMe.exists() && testMe.list().length > 0) {
```

Comment:
The problem in this line is not that `testMe` could be null, it can’t because of line 456, but the problem is that the result of `testMe.list()` could be null, so querying `lenght` on the result of `list()` could cause a `NullPointerException`.